src/inspirehep-holdingpen-js/holdingpen/holdingpen.controllers.js

```diff
@@ -78,7 +78,8 @@
       HoldingPenRecordService.setBatchDecision($scope.vm.invenioSearchResults.hits.hits, [+id], decision);
     }
     function hasConflicts(record) {
-      return record._source._extra_data.conflicts !== undefined && record._source._extra_data.conflicts.length > 0;
+      var _extra_data =  record._source._extra_data;
+      return _extra_data && _extra_data+.conflicts !== undefined && _extra_data.conflicts.length > 0;
```

Comment:
this line is invalid, you can also see travis long for what is wrong,  @iulianav.

I strongly suggest running linter locally by configuring your code editor to do it auto, after file save or sth.